portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,73 @@
 <!DOCTYPE html>
 <html>
+    
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- <style>
+         body {
+            background-image: url("/images.JPG");
+         }
+      </style> -->
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <!-- <img class = bgImage src = "/images/bg2.JPG"> -->
+       
+      <button id="play_button" onclick="loadIntro() ">Cynthia Ma</button>
+      <!-- <div id="greeting-container"></div> -->
+
+    <h1 id="introTitle" style="display:none">Cynthia's Portfolio</h1>
+      <p  id="introInfo" style="display:none">Hi I'm Cynthia. I'm a sophomore studying Computer Science and Statistics at Cornell University. 
+          I like photography and playing tennis and traveling. I was born and raised in New York and spend most of my time 
+          taking pictures, hanging out with my friends, or learning new things. I like to read and do my best to keep reading even 
+          when life gets hectic. 
+      </p>
+
+      <button id="profesh1" class="profesh" onclick=" window.open('https://github.com/cynthia-m','_blank')" style="display:none">Github</button>
```

Comment:
Here you're using buttons as links, see if you can make this simpler using the "a" tag, and putting the link in the href.